src/content/events/2024/hackathon-march-2024/index.md

```diff
@@ -0,0 +1,85 @@
+start_time: '10:00+02:00'
+end_date: '2024-03-20'
+end_time: '18:00+02:00'
+location_name: Gather town, Slack, and local sites.
+import_typeform: true
+---
+
+# Welcome
+
+Join us online March 18-20, 2024, for the nf-core hackathon! 🗓️
+
+The nf-core hackathon is free and is being held online (we will be back in [Gather](https://gather.town/)).
+Some community members are also hosting [local sites](#local-events).
+
+:::info
+The event will be free and we welcome everyone!
+More information about registration coming soon.
+:::
+
+# Online: Gather
+
+The hackathon is being held in a virtual venue on the platform Gather. If you have never used Gather before we have [a bytesize video](https://nf-co.re/events/2022/bytesize-37-gathertown) from the March 2022 hackathon that will help you to get started.
+
+<!--
+To join, you will need to follow these steps:
+
+1. Follow [this link](https://app.gather.town/app/br78S294XBCL487A/nf-core-hackathon) to find the Gather space
+   - Please note that the hackathon space will not open until the event
+2. Enter the email address you used to register for the hackathon
+   - This is the same address from which you received this email
+3. Check your email for a one-time code and enter it into the Gather window
+4. Enter the space and enjoy the hackathon!
+-->
+
+# Local sites
+
+In March 2023 we tried something new: distributed local sites.
+It was a massive success, with many people volunteering to organise local groups and great feedback all around.
+In 2024 we hope to replicate and build on that success! 🚀
+
+Hacking in person is a lot of fun. Although the hackathon is online, to get the best of both worlds, we are encouraging everyone to create local sites where people can gather to hack together. Be it with your group, institute, city, or region, local sites will be a lot of fun!
+
+If you have registered for a local site your local host will be in contact with you to outline site-specific information.
+For the most recent information about your site as well as the contact details of your local site organizer please check the event page specific to your local sites.
+
+> Reminder: Even if you are attending a local site you will still join everyone online in Gather.
+
+:::info
+More information about organising a local site coming soon!
+:::
+
+<!--
+<div class="table-responsive" markdown="1">
+
+| Country         | City      |  Location                                 | Event page                                |
+| --------------- | --------- | ----------------------------------------- | ----------------------------------------- |
+| 🇧🇷 Brazil       | Natal     | Federal University of Rio Grande do Norte | [Read more](br-ufrn.md)                   |
+| 🇫🇷 France       | Rennes    | IGDR                                      | [Read more](france-igdr.md)               |
+| 🇩🇪 Germany      | Tübingen  | QBiC                                      | [Read more](germany-qbic.md)              |
```

Comment:
We might not organise a local event this year but collaborate with heidelberg instead. Need to clear this up first now that the dates are fixed :) 
```suggestion
```

Reply:
I think this is all commented out on the render. Will commit anyway and can add a Heidelberg site back in later once everything is cleared up.

Reply:
Yes instead of deleting I just commented this block as I figured we would be adding sites imminently. Could delete and copy from the 2023 event page I guess, but shouldn't matter either way 🤷🏻‍♂️

Reply:
my bad, didn't see the comment line above and was in a rush. all good from my side then :)